### DISCUSSION TITLE
Manually updating package versions.

### DISCUSSION THREAD

**Discussion by malecov** [Azure/aad-pod-identity#1395] [Q&A] [answered]
Good Day Everyone.
Would like to ask if it is possible to update the package versions inside the project to accommodate security alerts?
I am new to this stuff and was encountering security alerts for the following:
- CVE-2021-46848
- CVE-2022-4450
- CVE-2023-0215
- CVE-2023-0286

**Comment by aramase** [comment, marked as answer]
The next release `v1.8.15` is planned for this week. That'll contain the base image CVE patches that contain fixes.

**Reply by malecov** [reply]
Thank you for the prompt response and action. can't wait for the fix to be released.

**Reply by malecov** [reply]
Hello Anish,
Would like to ask if there are any news related to the next release?

**Reply by aramase** [reply]
There was a delay because of a bug I found with IMDS from the CI results. We are targeting a release this week or earliest by next week.

**Reply by aramase** [reply]
`v1.8.15` released: https://github.com/Azure/aad-pod-identity/releases/tag/v1.8.15
